modules/core/plugin-container/src/main/java/org/rhq/core/pc/inventory/InventoryManager.java

```diff
@@ -1945,7 +1945,8 @@ private Configuration mergeExistingResourcePluginConfiguration(Resource resource
             Property discoveredProp = pluginConfig.get(propertyName);
             // Only overrides the property if it is a read-only property
             // See BZ: 1379834
-            if (propDefs.get(propertyName).isReadOnly()) {
+            PropertyDefinition propDefinition = propDefs.get(propertyName);
```

Comment:
It's never null. 

https://github.com/rhq-project/rhq/blob/master/modules/core/domain/src/main/java/org/rhq/core/domain/configuration/definition/ConfigurationDefinition.java#L182